finality-aleph/src/network/base_protocol/config.rs

```diff
@@ -47,7 +46,7 @@ where
                 // The best block number, always send a dummy value of 0.
                 0,
                 // The best block hash, always an obviously dummy value.
```

Comment:
Maybe remove this comment and replace the previous one with something like "We always pretend the genesis block is our best block."

Reply:
done